### PR TITLE
Move "role of IOTA" to its own page

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

I moved the `role of IOTA` to its own page. This topic will just get more important over time and I think it deserves its own
section :)

## Type of change

- Documentation Fix

### CODE REVIEW

File: internal/learn/future/assembly.md [hunk 1]
```diff
@@ -13,14 +13,4 @@ keywords:
...
-
-### The new dust protection
-
-With the new [dust protection](https://github.com/iotaledger/tips/pull/39), all data stored on the Tangle needs to be backed by IOTA Tokens. For example, every NFT, SC chain state anchor or Native Asset requires a deposit of IOTA Tokens in order to be stored on L1. You can read more about the new dust protection in this [community blog post](https://medium.com/@wernerderchamp/dust-protection-on-the-iota-network-an-eli12-d8ca567a2d36).
-
-### Mana
-
-Assembly nodes have to anchor the chain state often, even if the Tangle is congested. To get a fair share of the Tangle throughput in the [next version](https://v2.iota.org) of the IOTA protocol, you will need [Mana](/IOTA-2.0-Research-Specifications/5.3Mana), and Mana is derived directly from the IOTA Token. Assembly nodes will have to hold IOTA tokens, which will increase demand even more.
+There is also an article about the [role of IOTA](./role-of-iota) which is really important for a better understanding why IOTA needs the IOTA token.
```
Comment (aleksei-korolev) [inline internal/learn/future/assembly.md:16]:
```suggestion
```

This is the next page in ToC, there is a button right under this paragraph anyway.

**Review by aleksei-korolev — APPROVED** [review]

File: internal/learn/future/assembly.md (hunk 1 above)
Comment (Dr-Electron) [reply internal/learn/future/assembly.md:16]: Ohhhh, true 😅

**Review by aleksei-korolev — APPROVED** [review]

**Comment by aleksei-korolev** [comment]
I don't have write access.

**Comment by Dr-Electron** [comment]
sacre bleu 😆

File: internal/learn/future/role-of-the-iota-token.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - smart contracts
+  - permissionless
+  - ASMB
+---
+
+With the introduction of [Assemly](./assembly) it's important to understand the role of the IOTA token. Assembly is build on top of the Tangle, it will need the Tangle for anchored state updates, interoperability between chains, for Native Assets and general access to a congested network. There are two main usages of the IOTA Token in Assembly and everyting build on top of IOTA.
+
+## The new dust protection
+
+With the new [dust protection](https://github.com/iotaledger/tips/pull/39), all data stored on the Tangle needs to be backed by IOTA Tokens. For example, every NFT, SC chain state anchor or Native Asset requires a deposit of IOTA Tokens in order to be stored on L1. You can read more about the new dust protection in this [community blog post](https://medium.com/@wernerderchamp/dust-protection-on-the-iota-network-an-eli12-d8ca567a2d36) or in [this](./dust-protection#new-tokenisation-framework) wiki articel about the history of the dust protection in IOTA.
```
Comment (jlvandenhout) [inline internal/learn/future/role-of-the-iota-token.md:15]:
```suggestion
With the new [dust protection](https://github.com/iotaledger/tips/pull/39), all data stored on the Tangle needs to be backed by IOTA tokens. For example, every NFT, SC chain state anchor or Native Asset requires a deposit of IOTA tokens in order to be stored on L1. You can read more about the new dust protection in this [community blog post](https://medium.com/@wernerderchamp/dust-protection-on-the-iota-network-an-eli12-d8ca567a2d36) or in [this Wiki article](./dust-protection#new-tokenisation-framework) about the history of the dust protection in IOTA.
```

File: internal/learn/future/role-of-the-iota-token.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## Mana
+
+Assembly nodes have to anchor the chain state often, even if the Tangle is congested. To get a fair share of the Tangle throughput in the [next version](https://v2.iota.org) of the IOTA protocol, you will need [Mana](/IOTA-2.0-Research-Specifications/5.3Mana), and Mana is derived directly from the IOTA Token. Assembly nodes will have to hold IOTA tokens, which will increase demand even more.
```
Comment (jlvandenhout) [inline internal/learn/future/role-of-the-iota-token.md:19]:
```suggestion
Assembly nodes have to anchor the chain state often, even if the Tangle is congested. To get a fair share of the Tangle throughput in the [next version](https://v2.iota.org) of the IOTA protocol, you will need [Mana](/IOTA-2.0-Research-Specifications/5.3Mana), and Mana is derived directly from the IOTA token. Assembly nodes will have to hold IOTA tokens, which will increase demand even more.
```

**Review by Critical94 — APPROVED** [review]

**Review by jlvandenhout — CHANGES_REQUESTED** [review]
Just some editorial changes.

**Review by jlvandenhout — APPROVED** [review]

**Pull request merged**